<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SampleToolBar</class>
 <widget class="QWidget" name="SampleToolBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>37</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Font Size:</string>
     </property>
     <property name="buddy">
      <cstring>liveSize</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="liveSize">
     <property name="maximum">
      <double>999.990000000000009</double>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="sampleButton">
     <property name="text">
      <string>Samples</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
     <property name="arrowType">
      <enum>Qt::UpArrow</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="opentypeButton">
     <property name="text">
      <string>OpenType</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
     <property name="arrowType">
      <enum>Qt::UpArrow</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QComboBox" name="languageCombo">
     <property name="toolTip">
      <string>Depending of the font, some scripts 
are especially supported if processed 
through some logic (ICU here).</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Shaping:</string>
     </property>
     <property name="buddy">
      <cstring>languageCombo</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
